README.md

```diff
@@ -0,0 +1,114 @@
+E](https://en.wikipedia.org/wiki/H%26E_stain) image using a reference image.  The module is in C++ and is also packaged
+for [Python](https://www.python.org/).
+
+H & E ([hematoxylin](https://en.wikipedia.org/wiki/Haematoxylin) and [eosin](https://en.wikipedia.org/wiki/Eosin)) are
+stains used to color parts of cells in a histological image, often for medical diagnosis.  Hematoxylin is a compound
+that stains cell nuclei a purple-blue color.  Eosin is a compound that stains extracellular matrix and cytoplasm pink.
+However, the exact color of purple-blue or pink can vary from image to image, and this can make comparison of images
+difficult.  This routine addresses the issue by re-coloring one image (the first image supplied to the routine) using
+the color scheme of a reference image (the second image supplied to the routine).
+
+Structure Preserving Color Normalization is a technique described in [Vahadane et al.,
+2016](https://doi.org/10.1109/TMI.2016.2529665) and modified in [Ramakrishnan et al.,
+2019](https://arxiv.org/abs/1901.03088).  The idea is to model the color of an image pixel as something close to pure
+white, which is reduced in intensity in a color-specific way via an optical absorption model that depends upon the
+amounts of hematoxylin and eosin that are present.  [Non-negative matrix
+factorization](https://en.wikipedia.org/wiki/Non-negative_matrix_factorization) is used on each analyzed image to
+simultaneously derive the amount of hematoxylin and eosin stain at each pixel and the image-wide effective colors of
+each stain.
+
+The implementation in ITK accelerates the non-negative matrix factorization by choosing the initial estimate for the
+color absorption characteristics using a technique mimicking that presented in [Arora et al.,
+2013](http://proceedings.mlr.press/v28/arora13.html) and modified in [Newberg et al.,
+2018](https://doi.org/10.1371/journal.pone.0193067).  This approach finds a good solution for a non-negative matrix
+factorization by first transforming it to the problem of finding a convex hull for a set of points in a cloud.
+
+The lead developer of InsightSoftwareConsortium/ITKColorNormalization is [Lee Newberg](https://github.com/Leengit/).
+
+## Installation for Python
+
+ITKColorNormalization and all its dependencies can be easily installed with [Python
+wheels](https://blog.kitware.com/itk-is-on-pypi-pip-install-itk-is-here/).  Wheels have been generated for macOS, Linux,
+and Windows and several versions of Python, 3.5, 3.6, 3.7, and 3.8.  If you do not want the installation to be to your
+current Python environment, you should first create and activate a [Python virtual environment
+(venv)](https://docs.python.org/3/tutorial/venv.html) to work in.  Then, run the following from the command-line:
+
+    pip install itk-spcn
+
+Launch `python`, import the itk package, and set variable names for the input images
+
```

Comment:
Add triple backticks here for code formatting.

Reply:
Done.